### PR TITLE
javascript-typing-tutor | Nick Hernandez

### PR DESCRIPTION
**Pull request by Nickolas-Hernandez**
Screen Shot 2021-02-02 at 7 58 51 PM

Is this AI yet??

### CODE REVIEW

File: javascript-typing-tutor/main.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    if (counter !== 0) {
+      $characters[counter].className = '';
+      counter--;
+      $characters[counter].className = 'current-character';
+    }
+  } else if (event.key === char.charAt(0)) {
+    $characters[counter].className = 'correct';
+    counter++;
+    correctEntries++;
+    if (counter === 30) {
```
Comment (thebearingedge) [inline javascript-typing-tutor/main.js:21]: This is a very specific number. We call them "magic" numbers. Like "why 30?". How about `counter === $characters.length`?

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Really fantastic. Would love to see a GIF or more screenshots. Also, you can make a small improvement to your JavaScript.

File: javascript-typing-tutor/main.js (hunk 1 above)
Comment (Nickolas-Hernandez) [reply javascript-typing-tutor/main.js]: https://user-images.githubusercontent.com/75342275/106803908-89f9d800-6619-11eb-8b26-308c40aa11a8.mp4

Comment (Nickolas-Hernandez) [reply javascript-typing-tutor/main.js]: couldnt make a gif but heres an mp4!

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**